src/sst/core/simulation_impl.h

```diff
@@ -79,11 +79,6 @@ class StatisticOutput;
 class StatisticProcessingEngine;
 } // namespace Statistics
 
-namespace Serialization {
-class ObjectMap;
-} // namespace Serialization
-
-
```

Comment:
`ObjectMap` is declared in `SST::Serialization` here instead of `SST::Core::Serialization`, so it's an unused declaration which `clang-tidy` warned about.